### PR TITLE
You can see how much ammo the condor minigun has now. Also typo fix.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
Just a missing examine. Where are my tendies. 

## Changelog
:cl:
fix: You can now see the amount of ammo the condor minigun has by just looking.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/cas_plane_parts.dm [hunk 1]
```diff
@@ -145,6 +145,13 @@
 	///ref to the static weapon
 	var/obj/structure/dropship_equipment/weapon/static_weapon
 
+/obj/structure/caspart/minigun/examine(mob/user)
+	. = ..()
+	if(static_weapon.ammo_equipped)
+		static_weapon.ammo_equipped.show_loaded_desc(user)
+	else
+		to_chat(user, "It's empty.")
```
Comment (BraveMole) [inline code/game/objects/structures/cas_plane_parts.dm:153]: add a span class, like span = notice

File: code/game/objects/structures/cas_plane_parts.dm [hunk 2]
```diff
@@ -145,6 +145,13 @@
 	///ref to the static weapon
 	var/obj/structure/dropship_equipment/weapon/static_weapon
 
+/obj/structure/caspart/minigun/examine(mob/user)
+	. = ..()
+	if(static_weapon.ammo_equipped)
+		static_weapon.ammo_equipped.show_loaded_desc(user)
```
Comment (BraveMole) [inline code/game/objects/structures/cas_plane_parts.dm:151]: use the ?. operator, you don't need the if

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/cas_plane_parts.dm (hunk 2 above)
Comment (BraveMole) [inline code/game/objects/structures/cas_plane_parts.dm:151]: make an early return so you can remove the else, or even better, inline this

File: code/game/objects/structures/cas_plane_parts.dm (hunk 1 above)
Comment (lbnesquik) [reply code/game/objects/structures/cas_plane_parts.dm:153]: Other weapons don't have a notice

File: code/game/objects/structures/cas_plane_parts.dm (hunk 2 above)
Comment (lbnesquik) [reply code/game/objects/structures/cas_plane_parts.dm:151]: Not sure what this means, sorry.

Comment (TiviPlus) [reply code/game/objects/structures/cas_plane_parts.dm:151]: usually not worth the effort to change, but bravemole would be correct here
```
//They mean this
if(!x)
   return
if(!y)
  return
stuff()
//as opposed to 
if(x)
   if(y)
       stuff()

```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**